src/OSPSuite.UI/Views/SimulationRunAnalysisView.cs

```diff
@@ -0,0 +1,38 @@
+﻿using DevExpress.XtraLayout.Utils;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Presentation.Views;
+using OSPSuite.UI.Controls;
+using OSPSuite.UI.Extensions;
+
+namespace OSPSuite.UI.Views
+{
+   public partial class SimulationRunAnalysisView : BaseUserControl, ISimulationRunAnalysisView
+   {
+      private ISimulationRunAnalysisPresenter _presenter;
+
+      public SimulationRunAnalysisView()
+      {
+         InitializeComponent();
+         layoutControlItem2.TextVisible = true;
```

Comment:
Rename those controls. Thsi is not ok to have the default n gaming in code like this

Reply:
done

Reply:
@georgeDaskalakis I would appreciate it if you could review your PR before submitting them to me for correctness so that I do not waste to much time on triviality. Can we try from now on that you give it a first pass, add comments etc.. so that I can navigate in a PR nicely and focus on what matters? Thanks :)

Reply:
@msevestre sorry for this you are right, I will be more careful with things like that in the future